README.md

```diff
@@ -12,6 +12,9 @@ Ultima Online game server, developed in C++.
 | :---: |
 | [![Discord Shield](https://discordapp.com/api/guilds/354358315373035542/widget.png?style=shield)](https://discord.gg/ZrMTXrs) |
 
+## Getting Started tutorial
+If you're new to the Sphere server and want to setup your first shard, <a href="https://github.com/Sphereserver/Source-X/blob/master/docs/Getting-started.md">this is your go-to guide</a>!
```

Comment:
Change to use local reference, instead of url

`[this is your go-to guide](docs/Getting-started.md)`

Reply:
It didn't work because I was using the wrong syntax :man_facepalming: . Fixed.